mobile/src/main/res/values/strings.xml

```diff
@@ -19,6 +19,7 @@
     <string name="today_celebrates_one">%1$s</string>
     <string name="today_celebrates_two">%1$s and %2$s</string>
     <string name="today_celebrates_many">%1$s and %2$d others</string>
+    <string name="age_today">%1$s   %2$d today</string>
```

Comment:
The app uses a "Turns X" (see string `"turns_age"`) when communicating birthdays with age. 
Instead of using a new string, I would prefer if you use the existing one instead and add the name of the contact programmatically.